client/src/app/sidebar/sidebar.component.css

```diff
@@ -0,0 +1,112 @@
+html, body {
```

Comment:
No need to declare these in html, body.

Can just declare the classes directly.

Reply:
done